src/DetailsView/components/details-view-content.tsx

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { ScanMetadata } from 'common/types/store-data/unified-data-interface';
+import { DetailsViewOverlay } from 'DetailsView/components/details-view-overlay/details-view-overlay';
+import { InteractiveHeader } from 'DetailsView/components/interactive-header';
+import { DetailsViewBody } from 'DetailsView/details-view-body';
+import { DetailsViewContainerProps } from 'DetailsView/details-view-container';
+import * as React from 'react';
+
+export type DetailsViewContentProps = DetailsViewContainerProps;
+
+export const DetailsViewContent = NamedFC<DetailsViewContentProps>('DetailsViewContent', props => {
+    const renderHeader = () => {
+        const storeState = props.storeState;
+        const visualizationStoreData = storeState ? storeState.visualizationStoreData : null;
```

Comment:
Do we need the store state checks when this is within details-view-container, that checks for all store having state?